pubs/apis.py

```diff
@@ -161,12 +160,6 @@ def arxiv2bibtex(arxiv_id, try_doi=True, ui=None):
         raise ReferenceNotFoundError(msg)
 
     entry = feed.entries[0]
-    if 'arxiv.org/api/errors' in entry['id']:  # server is returning an error message.
-        msg = 'the arXiv server returned an error message: {}'.format(entry['summary'])
-        raise ReferenceNotFoundError(msg)
```

Comment:
@omangin: Just so you understand why I removed this. I talked with the arXiv devs. This would have generated a 400 HTTP status code, and already been handled above.